MpsAllocatorSample/Program.cs

```diff
@@ -0,0 +1,212 @@
+﻿using PlayFab;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace MpsAllocator
+{
+    class Program
+    {
+        static async Task Main(string[] args)
+        {
+            Console.WriteLine("Welcome to the MpsAllocatorSample!\nThis sample allows you to easily call frequently used APIs on your MPS Build");
+            Console.WriteLine("Enter TitleID");
+            PlayFabSettings.staticSettings.TitleId = Console.ReadLine();
+            
+            Console.WriteLine("Enter developer secret key");
+            PlayFabSettings.staticSettings.DeveloperSecretKey = Console.ReadLine();
```

Comment:
Just a general comment. I was pinged by security once when i has run a powershell script that has some secrets. They knew because they log commands that we run... Crazy! 

Anyway, this would promote someone hard coding the string. Maybe leave a comment saying please using something like keyvault or some secret store to retrieve. Overkill? Thoughts?

Reply:
thanks, I'll leave a comment, yeah. Maybe we can extend it to use ENV in a future iteration :)